src/Microsoft.OData.Core/ODataDeltaDeletedEntry.cs

```diff
@@ -4,6 +4,10 @@
 // </copyright>
 //---------------------------------------------------------------------
 
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;

```

Comment:
should we move these usings inside the namespace scope?

Reply:
yes.

---
In reply to: [150156059](https://github.com/OData/odata.net/pull/989#discussion_r150156059) [](ancestors = 150156059)